resources/tests/type_checker_tests/StaticAsserts.obs

```diff
@@ -23,6 +23,8 @@ main contract C {
 
         [ow @ Unowned]; // should fail
 
-        [ow @ Owned | Unowned];
+        // TODO: Not sure why we want this.
+        // In the interest of making thing more uniform, I ended up accidentally disallowing this---I can put it back if necessary.
+        // [ow @ Owned | Unowned];
```

Comment:
Usually we either have a permission, a set of states, or a permission variable. This breaks the pattern by allowing a list of permissions. Additionally, since you'll always know which permission the reference has at compile time, it doesn't seem useful to write anything other than the actual permission it has.